.pre-commit-config.yaml

```diff
@@ -33,3 +33,17 @@ repos:
       - --license-filepath
       - scripts/LICENSE-template.txt
       - --fuzzy-match-generates-todo
+- repo: https://github.com/wilu-nordic/pre-commit-sphinx
```

Comment:
I think this should be moved to the general Nordic account.

Reply:
@wilu-nordic @masz-nordic any idea what to do with this?

Reply:
This can stay for now.
@wilu-nordic can you send a PR to original repo so we don't have to maintain separate fork?

Reply:
@masz-nordic I will do it, but I need to add one more feature with saving warnings to the file - as it is done manually in nrfx repository